lib/elixir/lib/application.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Application do
   @moduledoc """
-  A module for working with applications and defining application callbacks.
+  Convenience functions for working with applications and defining
```

Comment:
hahaha. that so true.
how about removing the "module thing" and leaving the rest as it was.
there are also a few changes about split long sentences. like the one in kernel.. stufff that shouldb't be in the summary